src/applications/my-education-benefits/reducers/index.js

```diff
@@ -22,12 +23,17 @@ export default {
   form: createSaveInProgressFormReducer(formConfig),
   data: (state = initialState, action) => {
     switch (action.type) {
+      case FETCH_PERSONAL_INFORMATION:
```

Comment:
Set flag that indicates a fetch is already in progress so we don't start a second one.